### DISCUSSION TITLE
channel reconnect notice

### DISCUSSION THREAD

**Discussion by RynTurtle** [Chatterino/chatterino2#4615] [Ideas]
I would like to know when I have connected to a specific channel  on reconnect. When twitch forces you to reconnect you get a reconnected message but I have no way of knowing when im actually able to talk/see the messages in a specific channel.  I have normally just closed my chatterino and re opened whenever I get the notice. 

I believe my problem is that its to do with channels but to be clear what I experience  regularly is a reconnection notice, me trying to type and not being able to see my own messages so I just have started to close and reopen. I do have a lot of channels open which is probably the cause of the slow cooldown because of twitch, seeing a "reconnected" makes me think that its all ready.

**Comment by Felanbird** [comment]
Under normal circumstances you can confirm you have reconnected to the channel due to the the message history that you missed loading, but in the case of Twitch requested reconnects, message history is also disconnected so this functionality is unavailable.

But I do agree this should be handled better.
